packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx

```diff
@@ -18,28 +18,63 @@ limitations under the License.
 
 import * as React from 'react'
 import { useCMS } from 'tinacms'
-import { InlineField } from 'react-tinacms-inline'
+import { InlineField, FocusRing, useInlineForm } from 'react-tinacms-inline'
+import { FocusRingStyleProps } from 'react-tinacms-inline/src/styles'
 import { Wysiwyg } from '../components/Wysiwyg'
 import { EditorProps } from '../types'
 
 export interface InlineWysiwygFieldProps extends Omit<EditorProps, 'input'> {
   name: string
   children: any
+  focusRing?: boolean | FocusRingStyleProps
```

Comment:
Does it work if you pass `true`? Curious just cause the block interface is `false | FocusRingStyleProps` so we should update that one as well if were going to have it accept any boolean

Reply:
I can test this also if you're looking for another pair of eyes

Reply:
Yeah I'm working on standardizing this on a boolean since you may want to enable a focusRing on something without specifying any custom options. Though that was designed around the WYSIWYG having the focusRing disabled by default, which I may change.